buddhi/README.md

```diff
@@ -9,7 +9,7 @@ Responsibilities:
 ## Usage
 
 ```shell
-docker run --rm quay.io/3scale/perftest-toolkit:buddhi-v1.0.0 -h
+docker run --rm quay.io/3scale/perftest-toolkit:buddhi-v1.1.0 -h
```

Comment:
Good catch!

Actually should be *latest*
```
docker run --rm quay.io/3scale/perftest-toolkit:buddhi-latest -h
```

Reply:
So I will change it to `latest`